ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -643,7 +643,7 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
-- as: aws-ipi-private-fips-f28-disasterrecovery
+- as: aws-ipi-private-fips-sdn-f28-disasterrecovery
```

Comment:
@geliu2016 so here you want to update the default network (ovn) to sdn?  is there another testing for ovn?

Reply:
for sdn from 4.15 and 4.16,  you may need to add one env `OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"`

Reply:
@zhaozhanqi as I sync with gpei, this parameter have been enabled in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/sdn/openshift-sdn-release-4.15.yaml#L95, is it exact?

Reply:
> @geliu2016 so here you want to update the default network (ovn) to sdn? is there another testing for ovn?

the default installation is ovn, exact? I just want to add coverage for sdn

Reply:
> @zhaozhanqi as I sync with gpei, this parameter have been enabled in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/sdn/openshift-sdn-release-4.15.yaml#L95, is it exact?

it's enables in developer's job.  however you need also add this parameter in your job profile

Reply:
> > @geliu2016 so here you want to update the default network (ovn) to sdn? is there another testing for ovn?
> 
> the default installation is ovn, exact? I just want to add coverage for sdn

Yes, default is OVN.  ok for me if this is for sdn

Reply:
I have many dr profiles, most of them cover ovn, and this one cover sdn.